integration-cli/cli/build/build.go

```diff
@@ -0,0 +1,30 @@
+package build
+
+import (
+	"strings"
+
+	icmd "github.com/docker/docker/pkg/testutil/cmd"
+)
+
+// WithDockerfile specify a dockerfile string to be used as Dockerfile for build operations
```

Comment:
@thaJeztah for now it's just operators but could be more 👼 (I want to move the `fakeContext` thingies here too)